src/routes/todos/+page.svelte

```diff
@@ -40,8 +40,8 @@
 		</ul>
 
 		<form action="?/addTodo" method="POST" use:enhance={addTodo}>
-			{#if form?.missing}
-				<div class="flex flex-row justify-between">
+			{#if !form?.success}
```

Comment:
I think this needs to be changed to ```js 	{#if form?.missing} ```, I think it's the thing causing the weird validation erros on page load.
```suggestion
				{#if form?.missing}
```